src/Symfony/Component/Process/Pipes/AbstractPipes.php

```diff
@@ -47,7 +47,9 @@ public function __construct($input)
     public function close()
     {
         foreach ($this->pipes as $pipe) {
```

Comment:
let's remove the foreach completly, php will cleanup handles automatically

Reply:
Not removing foreach completely then?

Reply:
not being too smart is usually safer :P